lib/restbase.js

```diff
@@ -164,6 +165,9 @@ RESTBase.prototype.defaultListingHandler = function(value, restbase, req) {
 
 // Special handling for external web requests
 RESTBase.prototype.defaultWebRequestHandler = function(req) {
+    // enforce the usage of UA
+    req.headers = req.headers || {};
+    req.headers['User-Agent'] = req.headers['User-Agent'] || this.rb_config.user_agent;
```

Comment:
All incoming header names are lowercased, however outgoing ones should be [RFC 7231-compliant](https://tools.ietf.org/html/rfc7231#section-5.5.3) (heh, notice the number of the RFC as being identical to the RB port :P). Hm, looking around now, it's true, we send out lowercased headers.

I agree consistency matters, but the larger question is: should we Correctly-Capitalise all of the outgoing headers? It's a standard to do so after all.